### PR TITLE
[9.0][IMP] account_payment_partner: Allow hide bank account in report invoice

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Comment by pedrobaeza** [comment]
IMO this should be done another way:

* In payment mode, don't require to put a bank account, via a third option (fixed, multiple or none), or directly making the field not required.
* I think this is already working for multiple bank accounts: in partner onchange. don't put any bank account on this case

File: account_payment_partner/views/account_payment_mode.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+<data>
```
Comment (rigo1985) [inline account_payment_partner/views/account_payment_mode.xml:8]: In version 10 "< data >" is only necessary when you use "noupdate". 
(For example: < data noupdate = "1" > )
You can omit it in this case.

File: account_payment_partner/views/account_payment_mode.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
```
Comment (rigo1985) [inline account_payment_partner/views/account_payment_mode.xml:2]: The copyright I think is not necessary in this file.

**Comment by luismontalba** [comment]
How can I chose this option? How can I hide the bank account in invoice report?

**Comment by pedrobaeza** [comment]
@sergio-teruel please tell me about my comments

**Comment by alexis-via** [comment]
I'm not sure to understand the generic need behind PR. Maybe the solution would be to change the onchange so that the field "partner_bank_id" stays empty on the invoice if the field 'bank_account_required' of account.payment.method is not true (or something like this....)

**Comment by pedrobaeza** [comment]
@alexis-via The idea here is to decide at payment mode level if the bank account in the invoice should be printed or not in the invoice report. There are a lot of cases when you don't want to print this bank account, and put other explanations in the payment mode description itself.

**Comment by pedrobaeza** [comment]
@sergio-teruel is this still needed for our projects or solution in v10 is enough?

**Comment by sergio-teruel** [comment]
There is not changes in v10, so it is needed, @rafaelbn, We need this?

**Comment by rafaelbn** [comment]
Please this is ready yo merge!!!  :+1: Could we merge? This is in production for long time...



ping @carlosdauden  @cubells @pedrobaeza

**Review by rafaelbn — APPROVED** [review]
:+1:

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**